charts/prometheus-postgres-exporter/values.yaml

```diff
@@ -24,6 +24,17 @@ service:
   labels: {}
   annotations: {}
 
+podSecurityContext: {}
+  # runAsUser: 1001
+  # runAsGroup: 1001
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # allowPrivilegeEscalation: false
+  # capabilities:
```

Comment:
please check which options actualy work for podsecuritycontect.
"capabilities" for example does not.